recipes/r-checkpoint/meta.yaml

```diff
@@ -0,0 +1,96 @@
+    checkpoint allows you to install packages as they existed on CRAN on a specific
+    snapshot date as if you had a CRAN time machine. To achieve reproducibility, the
+    checkpoint() function installs the packages required or called by your project and
+    scripts to a local library exactly as they existed at the specified point in time.
+    Only those packages are available to your project, thereby avoiding any package
+    updates that came later and may have altered your results. In this way, anyone using
+    checkpoint's checkpoint() can ensure the reproducibility of your scripts or projects
+    at any time. To create the snapshot archives, once a day (at midnight UTC) Microsoft
+    refreshes the Austria CRAN mirror on the "Microsoft R Archived Network" server (<https://mran.microsoft.com/>).
+    Immediately after completion of the rsync mirror process, the process takes a snapshot,
+    thus creating the archive. Snapshot archives exist starting from 2014-09-17.
+  license_family: GPL2
+
+# The original CRAN metadata for this package was:
+
+# Package: checkpoint
+# Title: Install Packages from Snapshots on the Checkpoint Server for Reproducibility
+# Description: The goal of checkpoint is to solve the problem of package reproducibility in R. Specifically, checkpoint allows you to install packages as they existed on CRAN on a specific snapshot date as if you had a CRAN time machine. To achieve reproducibility, the checkpoint() function installs the packages required or called by your project and scripts to a local library exactly as they existed at the specified point in time. Only those packages are available to your project, thereby avoiding any package updates that came later and may have altered your results. In this way, anyone using checkpoint's checkpoint() can ensure the reproducibility of your scripts or projects at any time. To create the snapshot archives, once a day (at midnight UTC) Microsoft refreshes the Austria CRAN mirror on the "Microsoft R Archived Network" server (<https://mran.microsoft.com/>). Immediately after completion of the rsync mirror process, the process takes a snapshot, thus creating the archive. Snapshot archives exist starting from 2014-09-17.
+# Version: 0.4.0
+# Date: 2017-04-10
+# Author: Microsoft Corporation
+# Maintainer: Andrie de Vries <adevries@microsoft.com>
+# Copyright: Microsoft
+# License: GPL-2
+# URL: https://github.com/RevolutionAnalytics/checkpoint
+# BugReports: https://www.github.com/RevolutionAnalytics/checkpoint/issues
+# Imports: utils
+# Depends: R(>= 3.0.0)
+# Suggests: knitr, rmarkdown, testthat(>= 0.9), MASS
+# VignetteBuilder: knitr
+# RoxygenNote: 6.0.1
+# NeedsCompilation: no
+# Packaged: 2017-04-12 12:20:17 UTC; adevries
+# Repository: CRAN
+# Date/Publication: 2017-04-12 13:32:01 UTC
+
+# See
+# http://docs.continuum.io/conda/build.html for
+# more information about meta.yaml
```

Comment:
can you also remove extra comments? https://bioconda.github.io/guidelines.html#comments-in-meta